README.md

```diff
@@ -81,7 +81,7 @@ graph width        | **Optional.** Graph width in pixel. Defaults to `640`.
 timerange          | **Optional.** Global time range for graphs. Defaults to `6h`.
 enableLink         | **Optional.** Enable/disable graph with a rendered URL to the Grafana dashboard. Defaults to `yes`.
 datasource         | **Required.** Type of the Grafana datasource (`influxdb`, `graphite` or `pnp`). Defaults to `influxdb`.
-defaultdashboard   | **Required.** Name of the default dashboard which will be shown for unconfigured graphs. **Important: `panelID` must be set to `1`!** Defaults to `icinga2-default`.
+defaultdashboard   | **Required.** Name of the default dashboard which will be shown for unconfigured graphs. Set to 'none' to hide the module output. **Important: `panelID` must be set to `1`!** Defaults to `icinga2-default`.
```

Comment:
Single quotes around `none` should be changed into backticks. This will render Markdown blocks similar to `panelID` above.